src/server.c

```diff
@@ -1962,7 +1962,7 @@ struct redisCommand redisCommandTable[] = {
      "",
      .subcommands=xgroupSubcommands},
 
-    {"xsetid",xsetidCommand,3,
+    {"xsetid",xsetidCommand,-3,
```

Comment:
It seems that `xsetid` is forgotten in redis-doc.

Reply:
Yep. This is "by the original design" as the command is only supposed to be used internally for replication. That said. I'm not sure it shouldn't have a man page with a proper warning saying don't use it :)

Reply:
i think we wanna document it (same as other commands invented for the purpose of AOF).

side note: we're gonna delete rewriteAppendOnlyFileRio and friends soon, which should make it unneeded, but i do think there are other tools that will want to use such commands.
e.g. a tool that parses an RDB file and translates it to a stream of commands.

Reply:
Resolved in redis-doc